### DISCUSSION TITLE
How to get Swell height

### DISCUSSION THREAD

**Discussion by hrabah** [NOAA-EMC/WW3#1312] [Q&A]
Hello,

I am using the latest version of WWII model and I can't find how to get parameters like: 

- Swell wave, and swell wave direction (I mean swell sperated from the signficant swell and wave height)
- Secondary wave height
Are these parameters available if yes which configuration should I use to get them.

Thank you for your support

**Comment by aliabdolali** [comment]
You can have all you mentioned from partitions either from point or gridded outputs. Look at ww3_ounf.nml(inp) and ww3_ounp.nml(inp). For gridded one, you need to define partititions in ww3_shel.nml(inp)/ww3_multi.nml(inp)

**Reply by hrabah** [reply]
Thanks @aliabdolali for the quick respons. Do you have any example that you can refer to.

**Reply by hrabah** [reply]
@aliabdolali would you please confirm if that is correct:

phs0 : is the height of wind wave
phs2 : is the secondary wave heigh

**Comment by hrabah** [comment]
I have tried to define partition of this variable : `PHS   Partitioned wave heights`
However, the result doesn't seem to be consistent, as you can see that a lot of data is missing from the phs0 field. 
Do you have any suggestion to solve this problem. 

`PHS0`
![image](https://github.com/user-attachments/assets/53829fa4-1b7f-464b-b6eb-4d3aa735bd61)
`HS`
![image](https://github.com/user-attachments/assets/0b03c829-7d30-4616-97e7-93e40d0a9325)

**Comment by mickaelaccensi** [comment]
this normal. PHS0 only shows wave heights related to wind effect. When the grid cell is only a swell partition, it won't be defined in the wind waves partitions. You can have a look at PHS1 to visualize the most energic swell partition (which also contains missing data for cells which are only wind waves or only secondary swell partition) and so on

Best

**Reply by hrabah** [reply]
Thanks @mickaelaccensi for you answer. Would you confirm that:
phs0 : is the height of wind wave
phs2 : is the secondary wave heigh

**Reply by mickaelaccensi** [reply]
yes
